### DISCUSSION TITLE
MOOSE INSTALLATION

### DISCUSSION THREAD

**Discussion by kwarteng10** [idaholab/moose#17602] [Q&A Getting Started] [answered]
I am trying to install MOOSE on my computer and I run into an error message when I tried downloading the  MOOSE conda packages, please find the error message below

An HTTP error occurred when trying to retrieve this URL.                                                                                                                                                                                     HTTP errors are often intermittent, and a simple retry will get you on your way.                                                                                                                                                             'https://conda.anaconda.org/idaholab/linux-64'

**Comment by milljm** [comment, marked as answer]
Did you try again?

We have no control over 'the cloud' where these files are stored. Anaconda at times can run into connectivity issues, and the error supplied is often the correct course of action (trying again... immediately).

**Comment by kwarteng10** [comment]
Yes I tried on my personal computer and its working fine
Thank you
